lib/util/git/private-key.spec.ts

```diff
@@ -10,39 +13,121 @@ jest.mock('fs-extra', () =>
     >('../../../test/fixtures')
     .fsExtra(),
 );
-jest.mock('../exec');
+jest.mock('../exec', () =>
+  jest
+    .requireActual<typeof import('jest-mock-extended')>('jest-mock-extended')
+    .mockDeep(),
+);
 
-const exec = mocked(exec_);
+const exec = mockedDeep(exec_);
 
 describe('util/git/private-key', () => {
   describe('writePrivateKey()', () => {
+    beforeEach(() => {
+      jest.resetAllMocks();
+      mockReset(exec);
```

Comment:
i think the reset above will already reset it?
```suggestion
```

Reply:
I think you're right, done: f4ec3f5a6c667a3378891921ce187bbd395ca97f